src/index.html

```diff
@@ -1,22 +1,438 @@
+      rel="stylesheet"
+    />
+    <link
+      rel="icon"
+      href="/src/images/favicon.png"
+    />
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="./style.scss"
     />
     <script
-      src="scripts/main.js"
+      src="./scripts/main.js"
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a
+              href="#"
+              class="top-bar__logo-link"
+            >
+              <img
+                src="./images/logo.png"
+                alt="MyBike logo"
+                class="top-bar__logo"
+              />
+            </a>
+
+            <div class="top-bar__icons">
+              <a
+                href="tel:+1 234 5555-55-55"
+                class="icon icon--phone"
+              ></a>
+              <a
+                href="#menu"
+                class="icon icon--menu"
+              ></a>
+            </div>
+          </div>
+
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+
+    <aside
+      class="page__menu menu"
+      id="menu"
+    >
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a
+            href="#"
+            class="top-bar__logo-link"
+          >
+            <img
+              src="./images/logo-black.png"
+              alt="MyBike logo"
+              class="top-bar__logo"
+            />
+          </a>
+          <a
+            href="#"
+            class="icon icon--close"
+          ></a>
+        </div>
+
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#"
+                >
+                  Home
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#about-us"
+                >
+                  About Us
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#compare"
+                >
+                  Compare Bikes
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#details"
+                >
+                  Details
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  class="nav__link"
+                  href="#contacts"
+                >
+                  Contacts
+                </a>
+              </li>
```

Comment:
Add an empty line after multiline sibling elements [Reference](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#:~:text=Add%20empty%20lines%20between%20multiline%20sibling%20blocks%20of%20HTML)

Reply:
Prettier configs, if you have any with "good practice" better then i'm already using, please send to me.